charts/grafana/Chart.yaml

```
@@ -1,6 +1,6 @@
-apiVersion: v1
+apiVersion: v2
 name: grafana
-version: 5.8.16
+version: 5.9.0
```

```suggestion
version: 6.0.0
```
Let's increment the major version for this.